lib/linguist/languages.yml

```diff
@@ -3635,6 +3644,13 @@ SourcePawn:
   tm_scope: source.sp
   ace_mode: text
 
+Spline Font Data:
```

Comment:
The [canonical name](https://fontforge.github.io/sfdformat.html) for the format is `Spline Font Database`, which is what this needs to be. Again, please make sure it's capitalised correctly (the way I've written it here).